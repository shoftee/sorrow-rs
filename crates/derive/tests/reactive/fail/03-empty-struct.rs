use sorrow_derive::Reactive;

#[derive(Reactive)]
pub struct EmptyStruct {}

fn main() {}
