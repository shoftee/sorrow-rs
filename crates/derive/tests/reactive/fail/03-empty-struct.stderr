error: derive(Reactive) is only supported on structs with one or more named fields.
 --> tests/reactive/fail/03-empty-struct.rs:4:24
  |
4 | pub struct EmptyStruct {}
  |                        ^^
